apps/storybook/src/Utilities.stories.mdx

```diff
@@ -184,9 +195,21 @@ const {
   ordinateScale,
   dataToWorld,
   worldToData,
+  worldToHtml
 } = useAxisSystemContext();
 ```
 
+| Name             | Description                                                                                                           | Type                                                  |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| `visSize`        | Visualization size (different from canvas size on ratio-constrained visualizations, like `HeatmapVis`)                | <code>Size</code>                                     |
+| `abscissaConfig` | Abscissa configuration object passed to `VisCanvas`                                                                   | <code>AxisConfig</code>                               |
+| `ordinateConfig` | Ordinate configuration object passed to `VisCanvas`                                                                   | <code>AxisConfig</code>                               |
+| `abscissaScale`  | Computes the X coordinate in the HTML space of a given abscissa value (or the opposite with `abscissaScale.invert()`) | <code>AxisScale</code>                                |
+| `ordinateScale`  | Computes the Y coordinate in the HTML space of a given ordinate value (or the opposite with `ordinateScale.invert()`) | <code>AxisScale</code>                                |
+| `dataToWorld`    | Converts a vector from data space to world space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
+| `worldToData`    | Converts a vector from world space to data space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
+| `worldToHtml`    | Converts a vector from world space to HTML space                                                                      | <code>(vec: Vector2 &#124; Vector3) => Vector2</code> |
```

Comment:
Even with backticks, I had to use this `&#124;` hack to prevent the pipes from being interpreted as column delimiters... Then I had to change the backticks to `<code>` to actually render the pipes and not `&#124`... And then I had to use `<code>` for the other types as well, because Storybook renders them differently than code wrapped in backticks... Go figure :disappointed: